2025 Team 08

MohawkAustinR opened the pull request.

MohawkRaphaelT commented:
Please do the following when able:
- Move the files from inside `maze1.zip` into their corresponding locations in unity-project
- Move the Markdown documents into `Assets/2024/Team13/`

RaphaelTetreault committed:
Add 2025 mini game screenshot for wiki

RaphaelTetreault committed:
Add parameter to override unity editor version in CI

michaelburt-mohawk committed:
Add music and attribution

michaelburt-mohawk committed:
Merge pull request #2 from michaelburt-mohawk/main

Add music and attribution

EvanC421 committed:
Create GDD

Scyth3Lucif3r committed:
Added Enemy Spawns, Player Models, UI, & Copied base Scene items

Scyth3Lucif3r committed:
Merge pull request #1 from Scyth3Lucif3r/Daniel_Scene_&_Asset_Branch

Added Enemy Spawns, Player Models, UI, & Copied base Scene items

EvanC421 committed:
Merge branch 'main' into Evan-Documentation/Design

EvanC421 committed:
Add basic enemy spawning function

Enemy spawning works, but there's no timers.

DylanBordynuik committed:
Changed Folder Hierarchy

Created team specific folder, moved our custom scene to new folder

EvanC421 committed:
Merge branch 'main' into Evan-Documentation/Design

DylanBordynuik committed:
Box colliders and placeholder script

Also created prefabs for player ships

DylanBordynuik committed:
Changed naming convention for prefabs

DylanBordynuik committed:
Update Survive_60_Seconds.unity

DylanBordynuik committed:
Added player controller scripts to player ships

DylanBordynuik committed:
Added Background

EvanC421 committed:
Add working timer to spawner

Enemy spawner 2 spawns enemies in the wrong direction. Working on fixing this soon.

EvanC421 committed:
Fix spawning issue

Enemies should be spawning normally now

DylanBordynuik committed:
Additional scene setup

added player controller script, applied bullet prefabs, players can move and shoot.

DylanBordynuik committed:
Added scripts, testing in unity scene

Created a script that changes the sprite of the player when shot, with a delayed respawn window. Player movement is also disabled during this respawn period

DylanBordynuik committed:
Comment edits and some logic changes to stop compile errors

Had the script named something that already existed and it was not happy

EvanC421 committed:
fully implement fast enemy and random enemy spawning

Add fast enemy that weaves between shots. Random spawning is slightly skewed so basic enemies have a higher chance of spawning.

EvanC421 committed:
Add heavy enemy

EvanC421 committed:
Fully add heavy enemy

EvanC421 committed:
Merge pull request #2 from Scyth3Lucif3r/Evan-Documentation/Design

Evan documentation/design

DylanBordynuik committed:
Merge branch 'Dylan-Player-Controller-Changes'

Scyth3Lucif3r committed:
Update GDD.md

Scyth3Lucif3r committed:
Update GDD.md
